### PR TITLE
custome schema and github docs update

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This PR includes the following updates:

- Custom schema integration
- Github docs addition
- Added CircleCi testing for disabled models
- Version upgrade to v0.2.0

### CODE REVIEW

**Review by fivetran-jamie — APPROVED** [review]
lgtm!

File: models/stg_intercom__team.sql [hunk 1]
```diff
@@ -1,5 +1,5 @@
---To disable this model, set the using_team_ variable within your dbt_project.yml file to False.
-{{ config(enabled=var('using_team', True)) }}
+--To disable this model, set the intercom__using_team_ variable within your dbt_project.yml file to False.
```
Comment (fivetran-jamie) [inline models/stg_intercom__team.sql:1]: lil typo 
```suggestion
--To disable this model, set the intercom__using_team variable within your dbt_project.yml file to False.
```

File: models/stg_intercom__team_admin.sql [hunk 2]
```diff
@@ -1,5 +1,5 @@
---To disable this model, set the using_team_admin variable within your dbt_project.yml file to False.
-{{ config(enabled=var('using_team', True)) }}
+--To disable this model, set the intercom__using_team_admin variable within your dbt_project.yml file to False.
```
Comment (fivetran-jamie) [inline models/stg_intercom__team_admin.sql:1]: should this be `intercom__using_team_admin` or `intercom__using_team`?

File: models/tmp/stg_intercom__team_admin_tmp.sql [hunk 3]
```diff
@@ -1,5 +1,5 @@
---To disable this model, set the using_team_admin variable within your dbt_project.yml file to False.
-{{ config(enabled=var('using_team', True)) }}
+--To disable this model, set the intercom__using_team_admin variable within your dbt_project.yml file to False.
```
Comment (fivetran-jamie) [inline models/tmp/stg_intercom__team_admin_tmp.sql:1]: same comment as before -- `intercom__using_team_admin` or `intercom__using_team`?

File: models/stg_intercom__team_admin.sql (hunk 2 above)
Comment (fivetran-joemarkiewicz) [reply models/stg_intercom__team_admin.sql:1]: Thanks! must have been a mistake from doing the massive renaming.

File: models/tmp/stg_intercom__team_admin_tmp.sql (hunk 3 above)
Comment (fivetran-joemarkiewicz) [reply models/tmp/stg_intercom__team_admin_tmp.sql]: team_admin was removed during dev and must have snuck its way in. Removed, thanks!

**Comment by fivetran-jamie** [comment]
looks good!

**Pull request merged**